modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -112,7 +112,28 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
       TableDialog.open(editor, true);
     }
   });
+
+  // Apply cell style using command (background color, border color, border style and border width)
+  // tinyMCE.activeEditor.execCommand('mceTableApplyCellStyle', false, { backgroundColor: 'red', borderColor: 'blue' })
+  // Remove cell style using command (an empty string indicates to remove the style)
+  // tinyMCE.activeEditor.execCommand('mceTableApplyCellStyle', false, { backgroundColor: '' })
+  editor.addCommand('mceTableApplyCellStyle', (_ui: boolean, args: Record<string, string>) => {
+    if (!Type.isObject(args)) {
+      return;
+    }
+    Obj.each(args, (value, style) => {
+      const formatName = 'tablecell' + style.toLowerCase().replace('-', '');
+      if (editor.formatter.has(formatName) && Type.isString(value)) {
+        // Remove the style if given an empty string
+        if (value === '') {
```

Comment:
I wasn't sure if given `null` as the value, whether that should also the remove the style or not?

Reply:
Yeah, I'm on the fence as well. I'm fine with what has been done here.